### DISCUSSION TITLE
Error with "Simple with logo" E-mail design when sending event-mails

### DISCUSSION THREAD

**Discussion by nilskamm** [pretix/pretix#2517] [Q&A]
We are using pretix 4.7.1 in Docker.

I want to use the E-mail design "Simple with logo" and use a different logo in both of my events. 
At the moment every mail gives me an error in the pretix log.
I can open the named files of both events with my browser and with wget from the server itself. 
Here is the error from the log file:
> [2022-03-08 09:57:13,573: INFO/MainProcess] Received task: pretix.base.services.mail.mail_send_task[a62dcd3e-6bdd-49d7-9f88-53b8036a9dae]
> [2022-03-08 09:59:23,076: ERROR/ForkPoolWorker-1] ERROR creating mime_image image_0[https://pretix.domain.net/media/pub/thumbs/85e5c90823d9f07071d37fa519da8d87.600_x120.png]
> Traceback (most recent call last):
>   File "/usr/local/lib/python3.9/site-packages/urllib3/connection.py", line 174, in _new_conn
>     conn = connection.create_connection(
>   File "/usr/local/lib/python3.9/site-packages/urllib3/util/connection.py", line 95, in create_connection
>     raise err
>   File "/usr/local/lib/python3.9/site-packages/urllib3/util/connection.py", line 85, in create_connection
>     sock.connect(sa)
> TimeoutError: [Errno 110] Connection timed out
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 703, in urlopen
>     httplib_response = self._make_request(
>   File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 386, in _make_request
>     self._validate_conn(conn)
>   File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 1040, in _validate_conn
>     conn.connect()
>   File "/usr/local/lib/python3.9/site-packages/urllib3/connection.py", line 358, in connect
>     conn = self._new_conn()
>   File "/usr/local/lib/python3.9/site-packages/urllib3/connection.py", line 186, in _new_conn
>     raise NewConnectionError(
> urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fa824d1f910>: Failed to establish a new connection: [Errno 110] Connection timed out
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/usr/local/lib/python3.9/site-packages/requests/adapters.py", line 440, in send
>     resp = conn.urlopen(
>   File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 785, in urlopen
>     retries = retries.increment(
>   File "/usr/local/lib/python3.9/site-packages/urllib3/util/retry.py", line 592, in increment
>     raise MaxRetryError(_pool, url, error or ResponseError(cause))
> urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='pretix.domain.net', port=443): Max retries exceeded with url: /media/pub/thumbs/85e5c90823d9f07071d37fa519da8d87.600_x120.png (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa824d1f910>: Failed to establish a new connection: [Errno 110] Connection timed out'))
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/pretix/src/pretix/base/services/mail.py", line 682, in convert_image_to_cid
>     response = requests.get(image_src, verify=verify_ssl)
>   File "/usr/local/lib/python3.9/site-packages/requests/api.py", line 75, in get
>     return request('get', url, params=params, **kwargs)
>   File "/usr/local/lib/python3.9/site-packages/requests/api.py", line 61, in request
>     return session.request(method=method, url=url, **kwargs)
>   File "/usr/local/lib/python3.9/site-packages/requests/sessions.py", line 529, in request
>     resp = self.send(prep, **send_kwargs)
>   File "/usr/local/lib/python3.9/site-packages/requests/sessions.py", line 645, in send
>     r = adapter.send(request, **kwargs)
>   File "/usr/local/lib/python3.9/site-packages/requests/adapters.py", line 519, in send
>     raise ConnectionError(e, request=request)
> requests.exceptions.ConnectionError: HTTPSConnectionPool(host='pretix.domain.net', port=443): Max retries exceeded with url: /media/pub/thumbs/85e5c90823d9f07071d37fa519da8d87.600_x120.png (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa824d1f910>: Failed to establish a new connection: [Errno 110] Connection timed out'))
> [2022-03-08 09:59:23,715: INFO/ForkPoolWorker-1] Task pretix.base.services.mail.mail_send_task[a62dcd3e-6bdd-49d7-9f88-53b8036a9dae] succeeded in 130.14086666800722s: None

The mails are sent but they take some minutes because of the timeout. 

Does anyone have an idea what i could check or do to solve this problem?

Thanks a lot!

**Comment by raphaelm** [comment]
Is pretix.domain.net your actual domain or did you just replace it in the log message? Is the URL to your system in pretix.cfg set correctly?

**Comment by nilskamm** [comment]
I replaced the actual domain for github.
In the pretix.cfg my domain is correct.
In the organizer settings i have set a custom domain which is tickets.domain.net.
In the event settings i don´t use a custom domain.
In the log file it shows me the domain from the pretix.cfg, not the one in the organizer settings. And this is also the domain i used while checking if i can open the files with my browser and from the server.

**Comment by kempert** [comment]
Hello, what you have changed in der config file?

I have the same issue and for the Pretix installation i have the Domain: pretix.domain.de and for the organizer ticket.domain.de.
Pretix is behind NGNIX Proxy Manger on a virtualised host.

Thank you.

**Comment by nilskamm** [comment]
I never got it running on my installation and just use the "standard" design. ¯\_(ツ)_/¯
